packages/hub/jest/test-setup.ts

```diff
@@ -3,6 +3,15 @@ import {GanacheServer} from '@statechannels/devtools';
 import {deploy} from '../deployment/deploy';
 
 export default async function setup() {
+  if (!process.env.NODE_ENV) {
+    throw Error(`The NODE_ENV env var must be set to "test"`);
+  }
+  if (process.env.NODE_ENV != 'test') {
+    throw Error(
+      `Run "yarn db:testSetup" and rerun the attempted script prefixed with "NODE_ENV=test"`
```

Comment:
I don't think we have defined this command yet.

Reply:
Also, with https://github.com/statechannels/monorepo/pull/695, tests pass with any NODE_ENV.

Reply:
Hmm so as of #693, should we just close this (and #697) then?

Reply:
Yeah I think so.